admin/src/components/CreationFormular.vue

```diff
@@ -129,21 +125,21 @@ export default {
       rangeMin: 0,
       rangeMax: 1000,
       selected: '',
+      userId: this.item.userId,
     }
   },
   methods: {
     updateRadioSelected(name) {
       // do we want to reset the memo everytime the month changes?
       this.text = this.$t('creation_form.creation_for') + ' ' + name.short + ' ' + name.year
       this.rangeMin = 0
-      this.rangeMax = name.creation
+      this.rangeMax = Number(name.creation)
```

Comment:
I stumbled over this function - main reason was the parameter `name` which is not correct in my opinion - I think it should be `selected` or `option`. It is a bit out of scope, but this refactoring would increase readability alot I think. If you don't want to do this, please just close.